modules/alloy/src/main/ts/ephox/alloy/api/ui/TieredMenu.ts

```diff
@@ -42,7 +42,9 @@ const tieredMenu: TieredMenuSketcher = single<TieredMenuSpecType, TieredMenuDeta
     Fields.onHandler('onRepositionMenu'),
     Fields.onHandler('onCollapseMenu'),
 
-    FieldSchema.defaulted('highlightImmediately', true),
```

Comment:
This is a breaking change... do we care? If we don't then we need to mark this as a major in `versions.txt`.